test/run-git.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+set -o verbose
+rm -rf /tmp/ops
+GIT="git -c user.email='foo@bar.com' -c user.name='foobar'"
+
+function tt {
+bundle exec ./bin/taste-tester test -s localhost -y -c ./test/tt-git.rb -v
```

Comment:
actually since that's the only thing i see here, I'll merge, but can you do  that in both tests when you do your next update, please?